src/features/deeplinks/typeGuard.ts

```diff
@@ -0,0 +1,5 @@
+import { AllowedDeeplinkRoutes, DEEPLINK_TO_SCREEN_CONFIGURATION } from './types'
+
+export const isAllowedRouteTypeguard = (routeName: string): routeName is AllowedDeeplinkRoutes => {
```

Comment:
```suggestion
export const isAllowedRouteTypeGuard = (routeName: string): routeName is AllowedDeeplinkRoutes => {
```
Si on veut être (preque) trop rigoureux ^^